### ISSUE TITLE
simple rules changes

### ISSUE THREAD

**Issue by sdtaylor** [weecology/bbs-forecasting#158]
I think 4 is basically saying to use more than just the worldclim dataset? If so how about this:

```
### Use more than just long term climate averages
The WorldClim dataset is one of the most commonly used sources of future climate conditions for ecological forecasts. While climate conditions can explain a large amount of variance in most ecological problems, other factors such as soil conditions, disturbance, and biotic interactions should not be ignored. This is especially true at small spatial scales [@mod2016].
```
I also think 5 is a bit vague. It could potentially be added to the above

**Comment by davharris** [comment]
@sdtaylor Was this fixed in #157, or would you like to go further (e.g. changing to something more like what you wrote above)?

**Comment by sdtaylor** [comment]
Yep, I'm happy with them now.
